### ISSUE TITLE
Interface 'Psr\Http\Client\ClientInterface' not found

### ISSUE THREAD

**Issue by codyjames** [diginov/craft-sentry-logger#3]
### Description
I'm getting an `Interface 'Psr\Http\Client\ClientInterface' not found` error after installing the plugin.

### Steps to reproduce

1. Install the plugin.
2. Make sure you aren't in dev mode or anything like that.
3. Try to view the site.

### Additional info

- Plugin version: 1.1.4
- Craft version:  3.6.11.2
- PHP version: 7.4.15
- Configuration method: Basic or Advanced (Same issue with both)

**Comment by martinleveille** [comment]
Hi @codyjames, can you send us your `composer.json` and `composer.lock` files at support@diginov.com ?

We can't reproduce this bug on a fresh Craft install, thanks.

**Comment by codyjames** [comment]
Yeah, I can get that over to you. Note that I'm also seeing the error `Fatal error: Uncaught Error: Closure object cannot have properties in /Users/cody/sites/noihsaf/vendor/yiisoft/yii2/log/Dispatcher.php:187`

I'll send over the composer stuff shortly.

**Comment by codyjames** [comment]
Note that this does happen when I try a different Sentry Craft plugin. I wonder if it has to do with the Sentry PHP SDK?

**Comment by martinleveille** [comment]
@codyjames This is exactly what I am currently working on. Trying to know which of your plugin force an old version of the curl client.

**Comment by martinleveille** [comment]
@codyjames I've figured out that your plugin `percipioglobal/craft-password-policy` use the package `dragonbe/hibp` that use an outdated dependency of `ricardofiorani/guzzle-psr18-adapter` at the version `1.0.3` that has been fixed recently with the version `1.0.4`.

A quick fix is to require the latest version of the `ricardofiorani/guzzle-psr18-adapter` to your packages to force the `1.0.4` version. You can run this command :

```
composer require ricardofiorani/guzzle-psr18-adapter "^1.0.4"
``` 

It would be interesting if you can open a ticket with the creator of this plugin to see if they can fix the problem. 

Thanks again for reporting the issue, it might help someone else!

**Comment by codyjames** [comment]
@martinleveille Thank you so much!
